tests/Tests/Unit/Transport/RestTransportTest.php

```diff
@@ -32,7 +32,6 @@
 
```

Comment:
Curious test on this file. Looks like ApiException has been removed from this file.

Reply:
The import statement isn't being used in the file so it's better to remove it. I imagine this got caught by a style checker